packages/playground/blueprints/src/lib/steps/common.ts

```diff
@@ -28,3 +24,31 @@ export async function updateFile(
 export async function fileToUint8Array(file: File) {
 	return new Uint8Array(await file.arrayBuffer());
 }
+
+/**
+ * Polyfill the File class in JSDOM which lacks arrayBuffer() method
+ *
+ * - [Implement Blob.stream, Blob.text and Blob.arrayBuffer](https://github.com/jsdom/jsdom/issues/2555)
+ *
+ * When a Resource (../resources.ts) resolves to an instance of File, the
+ * resulting object is missing the arrayBuffer() method in JSDOM environment
+ * during tests.
+ *
+ * Import the polyfilled File class below to ensure its buffer is available to
+ * functions like writeFile (./client-methods.ts) and fileToUint8Array (above).
+ */
+class FilePolyfill extends File {
```

Comment:
Ideally this would live in tests, but that makes importing Blueprints in other packages tricky as every package would have to polyfill it. I think this is a neat solution 👍